data/json/recipes/recipe_food.json

```diff
@@ -1907,7 +1907,7 @@
     "time": "1 h",
     "autolearn": true,
```

Comment:
```suggestion
```
Might also reduce or remove the Survival component here. I know a few cheesemakers and I don't think they are all great at wilderness skills.